src/qililab/qprogram/qprogram.py

```diff
@@ -185,28 +220,50 @@ def play(self, bus: str, waveform: Waveform | IQPair):
         operation = Play(bus=bus, waveform=waveform)
         self._active_block.append(operation)
 
-    def wait(self, bus: str, time: int):
+    @requires_domain("duration", Domain.Time)
+    def wait(self, bus: str, duration: int):
         """Adds a delay on the bus with a specified time.
 
         Args:
             bus (str): Unique identifier of the bus.
             time (int): Duration of the delay.
         """
-        operation = Wait(bus=bus, time=time)
+        operation = Wait(bus=bus, duration=duration)
```

Comment:
do we handle long_waits?

Reply:
What is long_wait?

Reply:
Ah! No, right now the maximum duration is 2^16 ns which is the maximum value of integers in Q1ASM. But you can always add two `wait` operations if you need more time.

Reply:
for instance if you execute a circuit and you want to wait for the relaxation time of the qubit before the next iteration, or if you have to wait for the remaining time of `repetition_duration` (which is typically 200_000) this will be more than the max time allowed by the wait instruction (from `qpysequence` because I cannot find this info on the qblox docs `INST_MAX_WAIT = 2**16 - 4`)

Reply:
Yes I agree. But `long_wait` in qpysequence's library is outdated. It forces the duration to be a multiple of 4 which is changed in the version of Qblox we use. Now, the durations can be a multiple of 1.

I will mark this as a future task, to support `long_wait` with one instruction, but for now there is the work around of adding multiple `wait` operations.

2^16ns however is plenty for relaxation times, anyway, right?